### ISSUE TITLE
v3.0.0 application has no icon on macOS 

### ISSUE THREAD

**Issue by claremacrae** [TheJaredWilcurt/UGUI-pa11y#58]
This isn't a show-stopper, but it does make it hard to spot Koa11y on the Doc.

macOS Catalina 10.15.1

When running, Koa11y is shown as an empty square on the dock:

<img width="150" alt="image" src="https://user-images.githubusercontent.com/4840096/70618805-a368e500-1c0b-11ea-8085-cb9e29ce83b4.png">

And in the Launchpad, it is the same:

<img width="262" alt="image" src="https://user-images.githubusercontent.com/4840096/70618881-c4313a80-1c0b-11ea-8b08-5b46f2eea645.png">

Same when Cmd+Tabbing through running programs...

Thank you for a great program, by the way!

**Comment by smth** [comment]
I wonder if the issue might be a missing size in the `app.icns` file. I found that the icon displays correctly when small (column view in Finder for example), but not when large (in the dock). Could perhaps be "Retina" specific.. Do you have a Retina display @claremacrae ?

**Comment by claremacrae** [comment]
Hi @smth thanks for the reply.

It's a 2019 MacBook Pro, so yes, Retina.

As an experiment, I've dropped the resolution to 1024 x 640 and there's still no icon - but I don't know what's going on behind the scenes, and whether it still counts as retina?

**Comment by smth** [comment]
I don't know either.. I'm pretty much just guessing here. Thought maybe if it was Retina thing, that might explain the problem going somewhat unnoticed.

**Comment by gwenf** [comment]
Yes, I am having the same issue on my Macbook pro screen. The icon doesn't appear in the menu at the bottom or in finder:
<img width="54" alt="Screen Shot 2020-04-03 at 12 15 50 PM" src="https://user-images.githubusercontent.com/10039233/78383631-b3d01180-75a6-11ea-8bb3-393b34d9e293.png">
<img width="111" alt="Screen Shot 2020-04-03 at 12 15 29 PM" src="https://user-images.githubusercontent.com/10039233/78383635-b5013e80-75a6-11ea-9b00-4ae6b7b2c793.png">

BUT, it does show up on my external monitor:
![Screen Shot 2020-04-03 at 12 27 40 PM](https://user-images.githubusercontent.com/10039233/78383683-c5b1b480-75a6-11ea-9174-0def609627bd.png)

**Comment by donni106** [comment]
I see something like a fallback icon on macOS Mojave 10.14.6?

<img width="104" alt="Bildschirmfoto 2020-04-06 um 11 08 40" src="https://user-images.githubusercontent.com/1942953/78542133-03a51780-77f7-11ea-89d9-dcf59ddcf5f0.png">

**Comment by cannandev** [comment]
I opened the app's Info in Finder. Then dragged the icon onto the Preview. Restarted the app and it now appears in the dock.
<img width="266" alt="Screen Shot 2020-04-21 at 5 20 51 PM" src="https://user-images.githubusercontent.com/381122/79915184-8384d180-83f4-11ea-8a1c-75ca79c51279.png">
<img width="322" alt="Screen Shot 2020-04-21 at 5 21 54 PM" src="https://user-images.githubusercontent.com/381122/79915242-9f887300-83f4-11ea-92b3-d52e33603645.png">

**Comment by donni106** [comment]
@cannandev I cannot drag images onto the preview. Are you on a more recent macOS version?
